src/routes/setting/setting.js

```diff
@@ -81,7 +81,10 @@ const form = ({
         break
     }
     let deprecatedSettings = setting.definition && setting.definition.type === 'deprecated'
-
+    // Convert * to a multiplication character that the browser can recognise
+    if (setting.id === 'guaranteed-engine-manager-cpu' && setting?.definition?.description) {
+      setting.definition.description = setting.definition.description.replaceAll('*', '&times;')
```

Comment:
Can we make this general instead not just for guaranteed CPU setting?

Reply:
I'm not sure if the other settings just want to use `*` to say something important.

Reply:
It should be fine, all characters in each description should be raw, so it's safe to escape *.

Reply:
<img width="296" alt="截屏2023-02-09 下午12 54 19" src="https://user-images.githubusercontent.com/27733391/217721458-97eec193-a6fb-4e9e-a995-eba445a0d42f.png">

An aggravated treatment such as the one shown above requires the `**~**` format. If &times;  is replaced globally it will not meet our expectations.

Reply:
The pr still having problems , the `Guaranteed Replica Manager CPU` setting also needs to be replaced, I'm trying to think of a better way to do this

Reply:
So that means we just regard each description as markdown-interpretable :thinking: 

Then, need to think about how to escape it from the raw string instead. `\*` will work?

Reply:
React mark-down not recognized `\*`, `\*` is not work. I thought of a way to match /\x20\*\x20/ig based on the reg, because if it were a normal format expression it would all be *xxx*, **xx**. The /\x20\*\x20/ig reg will match * itself, so global substitution will do what we want. I'll provide a demo environment for you to check in a moment.